File: 1450. Number of Students Doing Homework at a Given Time/Solution.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        
        count = 0
        
        for i, _ in enumerate(startTime):
            if queryTime <= endTime[i] and queryTime >= startTime[i]:
                count += 1
        
        return count
        

File: 1450. Number of Students Doing Homework at a Given Time/Solution.cpp
class Solution {
public:
    int busyStudent(vector<int>& startTime, vector<int>& endTime, int queryTime) {
        
        int i;
        int cnt = 0;
        
        for(i=0;i<startTime.size();i++)
        {
            if ((startTime.at(i) <= queryTime) && (endTime.at(i) >= queryTime))
            {
                cnt++;
            }
        }
        
        return cnt;
    }
};
